update/application.go

```diff
@@ -180,7 +180,7 @@ func (cmd *applicationCmd) applyUpdates(app *apps.Application) {
 
 	var env map[string]string
 	if cmd.Env != nil {
-		env = *cmd.Env
+		env = cmd.Env
```

Comment:
Can't we then get rid of the `env` variable and use `cmd.Env` directly? The `env` variable will also be `nil` if `cmd.Env` is `nil`.

Reply:
But we do this already. If `cmd.Env` is `nil` (which it is if it is not set, right?), then the `env` var is also `nil` (as we don't use `make`).

Reply:
Yeah you are right, we don't make the make there, we just do it for the `buildEnv`.